packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -54,6 +64,47 @@ interface Props {
   siteInformationProps: SiteInformationProps;
 }
 
+type App = {
```

Comment:
**Question to benevolent reviewer(s)**: I wanted to get the type of `app`. This is copy/pasted from VS Code. What's the better way of doing this?

Reply:
Resolved by https://github.com/VEuPathDB/web-monorepo/pull/8/commits/73c5b41c7d1cfd0bed13bd0a782600c47a7a5fb2